### PR TITLE
libvisio 0.1.3

### PR DESCRIPTION
**Pull request by luzpaz**
removed dependencies on libwpd, libwpg
added dependencies for libxml2, gperf, zlib

### CODE REVIEW

**Comment by luzpaz** [comment]
@bfontaine resubmitted without the revision line

**Comment by luzpaz** [comment]
can someone give their blessing and commit this ? :) thx

File: Library/Formula/libvisio.rb [hunk 1]
```diff
@@ -15,10 +14,11 @@ class Libvisio < Formula
...
   depends_on "boost" => :build
   depends_on "cppunit" => :build
-  depends_on "libwpd"
-  depends_on "libwpg"
-  depends_on "icu4c"
   depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "libxml2"
+  depends_on "homebrew/dupes/gperf"
+  depends_on "homebrew/dupes/zlib"
```
Comment (tdsmith) [inline Library/Formula/libvisio.rb]: Can you explain these new dependencies and why the system version aren't useful?

Comment (luzpaz) [inline Library/Formula/libvisio.rb]: honestly, I don't know if using the dupes (gperf and zlib) are necessary for sure. What I do know is that they [are actual dependencies](https://wiki.documentfoundation.org/DLP/Libraries/libvisio#Building_the_code) and that from my experience using building an application using homebrew, Scribus, that I needed to use the zlib dupe. So I was playing it safe.

Comment (mikemcquaid) [inline Library/Formula/libvisio.rb]: Please don't add them unless it fails to compile.

Comment (luzpaz) [inline Library/Formula/libvisio.rb]: Good news: I removed them and it compiles. But I've only tested it for Mavericks. Is that an issue ?

Comment (mikemcquaid) [inline Library/Formula/libvisio.rb]: CI will test Yosemite/El Capitan.

Comment (luzpaz) [inline Library/Formula/libvisio.rb]: ok.. so libvisio can be compiled without homebrew's libxml2, gperf, libz 
the icu4c is not a new dep. It's in the current homebrew formula. 
I dont know how to squash commits well.. can you just squash them and commit ?

Comment (mikemcquaid) [inline Library/Formula/libvisio.rb]: They haven't been committed here. Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

Comment (luzpaz) [inline Library/Formula/libvisio.rb]: sorry @mikemcquaid i'm having some majors issues with git working. Do you mind do this?

Comment (mikemcquaid) [inline Library/Formula/libvisio.rb]: It's fine for you to just commit them unsquashed but I can't add commits to this PR.

**Comment by luzpaz** [comment]
ok.. will get to this tomorrow..

**Comment by luzpaz** [comment]
Just putting this here so I can address it soon
```
<dtardon> pass CXXFLAGS=-DBOOST_SYSTEM_NO_DEPRECATED to make  
<dtardon> it's a change in recent boost version(s)  
<dtardon> -DBOOST_ERROR_CODE_HEADER_ONLY might be needed too  
```

**Comment by bfontaine** [comment]
You’ll also need to rebase the PR on top of `master`.

**Comment by apjanke** [comment]
I rebased in #47126 and merged. The commits there preserve you as the author.

Thank you for your contribution to Homebrew!

**Pull request closed without merging**